sdk/react/src/DevCycleProvider.tsx

```diff
@@ -19,13 +19,15 @@ export function DevCycleProvider(props: Props): React.ReactElement {
     const { config } = props
     const { user, options } = config
     const [isInitialized, setIsInitialized] = useState(false)
+    const throwOnError =
+        options?.throwOnError !== undefined ? options.throwOnError : true
     let sdkKey: string
     if ('sdkKey' in config) {
         sdkKey = config.sdkKey
     } else {
         sdkKey = config.envKey
     }
-    if (!sdkKey) {
+    if (!sdkKey && !options?.throwOnError) {
```

Comment:
similar to previous, split this into a nested IF which will log an error if the key is missing but `throwOnError` is false